tests/lms/config/resources_test.py

```diff
@@ -40,6 +42,50 @@ def test_hypothesis_config_includes_grant_token(self, lti_launch):
         assert before <= claims["nbf"] <= after
         assert claims["exp"] > before
 
+    def test_hypothesis_config_includes_the_group(
+        self, lti_launch, models, pyramid_request
+    ):
+        group = models.CourseGroup.get.return_value
+
+        groups = lti_launch.hypothesis_config["services"][0]["groups"]
+
+        models.CourseGroup.get.assert_called_once_with(
+            pyramid_request.db,
+            mock.sentinel.tool_consumer_instance_guid,
+            mock.sentinel.context_id,
+        )
+        assert groups == [group.pubid]
+
+    def test_if_theres_no_tool_consumer_instance_guid(
```

Comment:
Can there be a case where there is neither a context_id nor a tool_consumer_instance_guid?

Reply:
Yep, could happen. I've added a test for it